src/Collection/utils.ts

```diff
@@ -24,6 +24,10 @@ export function getThirdPartyCollectionURN(
   return `${third_party_id}:${urn_suffix}`
 }
 
+export function isTPCollection(collection: CollectionAttributes): boolean {
```

Comment:
We can use this method on `upsertTPWCollection` too!

```ts
if (collection) {
if (
  collection.third_party_id === null ||
  collection.urn_suffix === null
) {
  throw new WrongCollectionException(
    "The collection can't be converted into a third party collection.",
    { id }
  )
}
```

Reply:
Used the `isTPCollection` function here.